cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -64,8 +64,6 @@ bool CobTwistController::initialize()
         return false;
     }
 
-    // Multi-Chain Support
-
```

Comment:
Can KinematicExtension be used to do something like combining the torso_chain and the arm_right_chain?